skins/oasis/modules/ContributeMenuController.class.php

```diff
@@ -36,6 +36,16 @@ public function executeIndex() {
 				'href' =>  $specialPageTitle->getLocalURL(),
 			];
 
+			/**
+			 * id for flow tracking
+			 * @see https://wikia-inc.atlassian.net/browse/WW-343
+			 */
+			if ( $specialPageName === 'CreatePage' ) {
+				$attrs[ 'id' ] = 'contribute-button-create-page';
+				// TODO: move param name and flow name  to consts
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)